api/net/Areas/Services/Controllers/ContentController.cs

```diff
@@ -252,6 +252,24 @@ public async Task<IActionResult> UpdateAsync(ContentModel model, bool index = fa
 
         return new JsonResult(new ContentModel(content));
     }
+    /// <summary>
+    /// Update content for the specified 'id'.
+    /// Will not trigger any re-index or audit trail update
+    /// </summary>
+    /// <param name="model"></param>
+    /// <param name="newStatus">New ContentStatus.</param>
+    /// <returns></returns>
+    [HttpPut("{id}/status")]
+    [Produces(MediaTypeNames.Application.Json)]
+    [ProducesResponseType(typeof(ContentModel), (int)HttpStatusCode.OK)]
+    [ProducesResponseType(typeof(ErrorResponseModel), (int)HttpStatusCode.BadRequest)]
+    [SwaggerOperation(Tags = new[] { "Content" })]
+    public IActionResult UpdateStatusAsync(ContentModel model, ContentStatus newStatus)
+    {
+        var content = _contentService.UpdateStatusOnly(model.Id, newStatus);
```

Comment:
I would probably lean to not bothering with a dedicated function for this.  I would just make a request for the entity in the db and change the status, then update.  But there can be benefits to do it your way.